test-e2e/NmriumPage.ts

```diff
@@ -76,4 +76,13 @@ export default class NmriumPage {
       this.page.locator('_react=FilterPanel >> text=Phase correction'),
     ).toBeVisible();
   }
+  public async checkSVGLength() {
```

Comment:
The idea here is still to hardcode in the test a minimal value and not base it on bodyLength.

This means that you need to first time to make a console.log to get the length of the SVG and then hardcode this value.

We could consider that the test pass if the value is >0.8 * expected value and <1.2 * expected value.